# `mymath.py`の`subtract_divide()`関数について2つのunittestテストケースを
# 記述します。
#
# 1.入力例から正しい結果が得られるか確認します
# 2.`CustomZeroDivisionError`が正しく発生することを確認します

